specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
```

Comment:
Even if I do `node app.js --files specs/data/transactions/may/transactions.csv`
my program will not search for the csv file in this path, instead will thing that this is the csv's name which is in the root directory, because I have this code in my app.js 

`if (isUserSpecified) {
        csvFile = fs.readFileSync(`./${file}`, "utf8");
    } else {
        csvFile = fs.readFileSync(`./files/${file}`, "utf8");
    }`